### PR TITLE
Cleanup examples/std/compression.go

### PR DESCRIPTION
**Pull request by cxiang03**
## Summary
Chores Example Codes Improve

## Checklist
- [x] Unit and integration tests covering the common scenarios were added
- [x] A human-readable description of the changes was provided to include in CHANGELOG

### CODE REVIEW

**Comment by jkaflik** [comment]
Thanks @cxiang03 . This will go with the next minor release.

File: examples/std/compression.go [hunk 1]
```diff
@@ -89,6 +91,7 @@ func CompressOpen() error {
 	if _, err := conn.Exec(`
 		CREATE TABLE example (
 			  Col1 Array(String)
+			, Col2 Uint64
```
Comment (jkaflik) [inline examples/std/compression.go:94]:
```suggestion
			, Col2 UInt64
```

**Review by jkaflik — CHANGES_REQUESTED** [review]
this introduces a bug

**Comment by cxiang03** [comment]
> this introduces a bug

🫣 applied

File: examples/std/compression.go [hunk 2]
```diff
@@ -49,6 +50,7 @@ func CompressOpenDB() error {
...
```
Comment (cxiang03) [inline examples/std/compression.go:53]:
```suggestion
			, Col2 UInt64
```

**Review by jkaflik — APPROVED** [review]

**Pull request merged**